opensearch/sycamore-opensearch.sh

```diff
@@ -226,6 +230,78 @@ END
     echo "EMBEDDING_MODEL_ID=${EMBEDDING_MODEL_ID}" >>"${PERSISTENT_ENV_TMP}"
 }
 
+sp_setup_reranking_model() {
+    local all_config=$(jq '@json' <<END
+{
+  "_name_or_path": "BAAI/bge-reranker-base",
+  "architectures": [
+    "XLMRobertaForSequenceClassification"
+  ],
+  "attention_probs_dropout_prob": 0.1,
+  "bos_token_id": 0,
+  "classifier_dropout": null,
+  "eos_token_id": 2,
+  "hidden_act": "gelu",
+  "hidden_dropout_prob": 0.1,
+  "hidden_size": 768,
+  "id2label": {
+    "0": "LABEL_0"
+  },
+  "initializer_range": 0.02,
+  "intermediate_size": 3072,
+  "label2id": {"LABEL_0": 0},
+  "layer_norm_eps": 1e-05,
+  "max_position_embeddings": 514,
+  "model_type": "xlm-roberta",
+  "num_attention_heads": 12,
+  "num_hidden_layers": 12,
+  "output_past": true,
+  "pad_token_id": 1,
+  "position_embedding_type": "absolute",
+  "torch_dtype": "float32",
+  "transformers_version": "4.33.3",
+  "type_vocab_size": 1,
+  "use_cache": true,
+  "vocab_size": 250002
+}
+END
```

Comment:
How often do you think we will change this model?  I assume most of these parameters are specific to the choice of the model?

Reply:
These params are def specific to the choice of model. On the other hand, they don't matter whatsoever. I've run tests with `"all_config": "nobody cares"`